extensions/vscode-api-tests/src/extension.ts

```diff
@@ -29,6 +30,7 @@ export function activate(context: vscode.ExtensionContext) {
 
 		vscode.commands.executeCommand('vscode.open', vscode.Uri.parse(`memfs:/sample-folder/large.ts`));
```

Comment:
@weinand @bpasero Can either of you explain why extension activation is disabled for node.js? Should my custom editor registration live somewhere else?

Reply:
@mjbvz I do not fully understand that question but the file you are linking to is only used for VSCode for Web to have a demo setup, e.g. used for https://vscode-web-test-playground.azurewebsites.net/?enter=true

Reply:
Please do not make changes to this file unless you want to support custom editors in web!

Reply:
To test custom editors end to end, I'd like to have a contribution point and a basic extension that registers a custom editor

I thought this this should live under `vscode-api-test` along with our other API tests (such as the webview tests). But it looks like that extension's activation is used for web and I don't understand why it is under `vscode-api-test`

Where should I to put my test custom editor extension instead? Do we have an other similar tests that create test extensions?

Reply:
@mjbvz yes it is the right location. Your test should be inside `extensions/vscode-api-tests/src/singlefolder-tests` and the activation goes into `extensions/vscode-api-tests/package.json`.

We should probably rename `extensions/vscode-api-tests/src/extension.ts` to `extensions/vscode-api-tests/src/THIS_FILE_IS_ONLY_USED_IN_WEB_PLAYGROUND.ts`, it is unrelated to running tests!

Reply:
I have moved `extension.ts` into a folder called `web-playground` to make it clearer.